consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/TransitionContext.java

```diff
@@ -139,4 +139,9 @@ public void putPayloadById(final PayloadIdentifier payloadId, final Block block)
   public Optional<Block> retrieveBlockById(final PayloadIdentifier payloadId) {
     return postMergeContext.retrieveBlockById(payloadId);
   }
+
+  @Override
+  public boolean isCheckpointPostMergeSync() {
+    return false;
```

Comment:
Shouldn't this delegate to the postMergeContext.isCheckpointPostMergeSync method?